ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -203,6 +203,160 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 12h
+  name: canary-openshift-origin-installer-e2e-azure-fips-4.2
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --secret-dir=/usr/local/pull-secret
+      - --secret-dir=/usr/local/e2e-azure-fips-cluster-profile
+      - --target=e2e-azure-fips
+      - --template=/usr/local/e2e-azure-fips
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      command:
+      - ci-operator
+      env:
+      - name: BRANCH
+        value: "4.2"
+      - name: CLUSTER_TYPE
+        value: azure4
+      - name: ENABLE_FIPS
+        value: "true"
+      - name: CONFIG_SPEC
+        value: |
+          tag_specification:
+            name: "$(BRANCH)"
+            namespace: ocp
+          resources:
+            '*':
+              limits:
+                memory: 4Gi
+              requests:
+                cpu: 100m
+                memory: 200Mi
+          tests:
+          - as: e2e-$(CLUSTER_TYPE)-fips
+            commands: TEST_SUITE=openshift/conformance/parallel run-tests
+            openshift_installer:
+              cluster_profile: "$(CLUSTER_TYPE)"
+          - as: e2e-$(CLUSTER_TYPE)-fips-serial
```

Comment:
Kind of annoying but in addition to adding the entry here in `tests` you actually need to dupe the entire job, add the `-serial` bit to the name and update the TEST_COMMAND to get a serial suite